<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1136</width>
    <height>927</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>1141</width>
     <height>931</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1141</width>
     <height>931</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1141</width>
     <height>931</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:'white';</string>
   </property>
   <widget class="QLabel" name="Title">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>0</y>
      <width>451</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.898, y2:0.977, stop:0.211454 rgba(4, 21, 132, 255), stop:1 rgba(5, 182, 191, 255));
font: 57 22pt &quot;Noto Sans&quot;;
border-bottom-left-radius:35px;
border-bottom-right-radius:35px;
</string>
    </property>
    <property name="text">
     <string>Recommend me a movie</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="searchBar">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>163</y>
      <width>371</width>
      <height>51</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Search Your Movie here.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Search bar.&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Search your movie.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit#searchBar{
color:black;
border:1px solid 'gray';
border-top-left-radius:25px;
border-bottom-left-radius:25px;
font: 57 16pt &quot;Noto Sans&quot;;
padding-left:25px;
}

QLineEdit#searchBar:focus{
border:1px dotted 'black';
}

QLineEdit#searchBar:hover{
background-color:qlineargradient(spread:pad, x1:0.506405, y1:1, x2:0.507, y2:0.0113636, stop:0 rgba(163, 163, 163, 145), stop:0.581498 rgba(255, 255, 255, 145));
}</string>
    </property>
    <property name="text">
     <string>Search Here</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="search">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>163</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Search button.&lt;br/&gt;Searches the movie having the matching name.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#search{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.898, y2:0.977, stop:0.211454 rgba(4, 21, 132, 255), stop:1 rgba(5, 182, 191, 255));
border:None;
border-top-right-radius:25px;
border-bottom-right-radius:25px;
}

QPushButton#search:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.898, y2:0.977, stop:0.211454 rgba(5, 182, 191, 255), stop:1 rgba(4, 21, 132, 255));
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>img/search.png</normaloff>img/search.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="next">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>280</y>
      <width>41</width>
      <height>261</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Next&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton#next{
border:None;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.898, y2:0.977, stop:0.211454 rgba(4, 21, 132, 255), stop:1 rgba(5, 182, 191, 255));
border-top-left-radius:15px;
border-bottom-left-radius:15px;
}

QPushButton#next:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.898, y2:0.977, stop:0.211454 rgba(5, 182, 191, 255), stop:1 rgba(4, 21, 132, 255));
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>img/right.png</normaloff>img/right.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>28</width>
      <height>28</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="moviePoster1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>290</y>
      <width>141</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(195, 195, 195);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="moviePoster2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>290</y>
      <width>141</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(195, 195, 195);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="moviePoster3">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>290</y>
      <width>141</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(195, 195, 195);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="moviePoster4">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>290</y>
      <width>141</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(195, 195, 195);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="movie1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>490</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(195, 195, 195);</string>
    </property>
    <property name="text">
     <string>Movie-1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="movie2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>490</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(195, 195, 195);</string>
    </property>
    <property name="text">
     <string>Movie-2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="movie3">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>490</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(195, 195, 195);</string>
    </property>
    <property name="text">
     <string>Movie-3</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="movie4">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>490</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(195, 195, 195);</string>
    </property>
    <property name="text">
     <string>Movie-4</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="prev">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>280</y>
      <width>41</width>
      <height>261</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Previous.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton#prev{
border:None;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.898, y2:0.977, stop:0.211454 rgba(4, 21, 132, 255), stop:1 rgba(5, 182, 191, 255));
border-top-right-radius:15px;
border-bottom-right-radius:15px;
}

QPushButton#prev:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.898, y2:0.977, stop:0.211454 rgba(5, 182, 191, 255), stop:1 rgba(4, 21, 132, 255));
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>img/left.png</normaloff>img/left.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>28</width>
      <height>28</height>
     </size>
    </property>
   </widget>
   <widget class="QWidget" name="recommended_moviesWidget" native="true">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>570</y>
      <width>991</width>
      <height>341</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.898, y2:0.977, stop:0.211454 rgba(4, 21, 132, 255), stop:1 rgba(5, 182, 191, 255));</string>
    </property>
    <widget class="QLabel" name="moviePoster">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>211</width>
       <height>281</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(13, 71, 122,50)</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="movieTitle">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>20</y>
       <width>611</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(13, 71, 122,50);
font: 57 20pt &quot;Noto Sans&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="movieRating">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>110</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(13, 71, 122,50)</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="movieDesc">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>130</y>
       <width>611</width>
       <height>191</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(13, 71, 122,50);
font: 25 15pt &quot;Noto Sans&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="movie5">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>490</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(195, 195, 195);</string>
    </property>
    <property name="text">
     <string>Movie-5</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="moviePoster5">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>290</y>
      <width>141</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(195, 195, 195);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="related_movies" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>270</y>
      <width>1141</width>
      <height>281</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(170, 170, 170,40);
border:None;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="help">
    <property name="geometry">
     <rect>
      <x>1061</x>
      <y>10</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans'; font-size:14pt; font-weight:56; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Help&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Help&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.898, y2:0.977, stop:0.211454 rgba(4, 21, 132, 255), stop:1 rgba(5, 182, 191, 255));
font: 57 14pt &quot;Noto Sans&quot;;
border:None;
border-radius:25px;
</string>
    </property>
    <property name="text">
     <string>?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rec_prev">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>690</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#rec_prev{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.898, y2:0.977, stop:0.211454 rgba(4, 21, 132, 255), stop:1 rgba(5, 182, 191, 255));
border:None;
border-radius:25px
}

QPushButton#rec_prev:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.898, y2:0.977, stop:0.211454 rgba(5, 182, 191, 255), stop:1 rgba(4, 21, 132, 255));
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>img/left-arrow.png</normaloff>img/left-arrow.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="rec_next">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>700</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#rec_next{background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.898, y2:0.977, stop:0.211454 rgba(4, 21, 132, 255), stop:1 rgba(5, 182, 191, 255));
border:None;
border-radius:25px
}

QPushButton#rec_next:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.898, y2:0.977, stop:0.211454 rgba(5, 182, 191, 255), stop:1 rgba(4, 21, 132, 255));
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>img/right-arrow.png</normaloff>img/right-arrow.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="header">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>98</y>
      <width>591</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;
font: 15 14pt &quot;Noto Sans&quot;;</string>
    </property>
    <property name="text">
     <string>We will recommend you 5 similar movies to your choice.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="shadow1">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>695</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:23px;
background-color:rgb(111, 111, 111);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="shadow2">
    <property name="geometry">
     <rect>
      <x>1084</x>
      <y>706</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:23px;
background-color:rgb(111, 111, 111);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="shadow3">
    <property name="geometry">
     <rect>
      <x>1065</x>
      <y>16</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:23px;
background-color:rgb(111, 111, 111);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="internetConnection">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>16</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>img/wifi.png</pixmap>
    </property>
   </widget>
   <widget class="QWidget" name="loading" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1141</width>
      <height>931</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0)</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>280</y>
       <width>371</width>
       <height>191</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:black;
background-color:white;
font: 25 italic 18pt &quot;Noto Sans&quot;;
border-radius:90px</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <zorder>loading</zorder>
   <zorder>shadow3</zorder>
   <zorder>shadow2</zorder>
   <zorder>shadow1</zorder>
   <zorder>related_movies</zorder>
   <zorder>Title</zorder>
   <zorder>searchBar</zorder>
   <zorder>search</zorder>
   <zorder>next</zorder>
   <zorder>moviePoster1</zorder>
   <zorder>moviePoster2</zorder>
   <zorder>moviePoster3</zorder>
   <zorder>moviePoster4</zorder>
   <zorder>movie1</zorder>
   <zorder>movie2</zorder>
   <zorder>movie3</zorder>
   <zorder>movie4</zorder>
   <zorder>prev</zorder>
   <zorder>recommended_moviesWidget</zorder>
   <zorder>movie5</zorder>
   <zorder>moviePoster5</zorder>
   <zorder>help</zorder>
   <zorder>rec_prev</zorder>
   <zorder>rec_next</zorder>
   <zorder>header</zorder>
   <zorder>internetConnection</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
